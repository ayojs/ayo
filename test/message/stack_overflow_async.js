// Flags: --stack_trace_limit=3

'use strict';
require('../common');

async function f() {
  await f();
}

async function g() {
  try {
    await f();
  } catch (e) {
    console.log(e);
  }
}

g();
