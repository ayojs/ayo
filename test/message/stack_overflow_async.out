RangeError: Maximum call stack size exceeded
    at f (*test*message*stack_overflow_async.js:*)
    at f (*test*message*stack_overflow_async.js:7:*)
    at f (*test*message*stack_overflow_async.js:7:*)
